### PR TITLE
remove library files, adjust composer.json

### PR DESCRIPTION
**Pull request by vakopian**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
         "psr-0": {
-            "Wsdl2PhpGenerator\\": ["src/", "lib/"]
+            "Wsdl2PhpGenerator\\": ["src/"]
         }
     },
     "bin": ["wsdl2php"],
     "require": {
         "php": ">=5.3.0",
         "ext-soap": "*",
+		"wsdl2phpgenerator/wsdl2phpgenerator": "3.*",
```
Comment (kasperg) [inline composer.json]: We use spaces instead on tabs for indentation.

File: composer.json [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
         "psr/log": "1.0.*",
         "symfony/console": "~2.3",
         "symfony/options-resolver": "~2.3"
```
Comment (kasperg) [inline composer.json]: `symfony/options-resolver` should be installed through the `wsdl2phpgenerator` library.

File: composer.lock [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
```
Comment (kasperg) [inline composer.lock:1]: For applications the `composer.lock` file should stay in the repository.

**Comment by kasperg** [comment]
Well done, @vakopian.

A few changes and this should be good to merge.

The README will need some updating but I will take care of that after merging.

File: composer.lock (hunk 3 above)
Comment (vakopian) [inline composer.lock:1]: I thought it was there. Is it not in the repository after this PR?

Comment (kasperg) [inline composer.lock:1]: My bad. I saw all the deletions and thought you had deleted `composer.lock` as well.

**Comment by kasperg** [comment]
Sorry. I was a little too quick to respond there.

The Travis build now fails due to a missing `bootstrap.php` referenced in [`phpunit.xml.dist`](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator-cli/blob/3.x/phpunit.xml.dist#L19). The file may no longer be needed so try removing the reference from the file and run the tests locally.

We no longer need PHPVCR in the project. You can remove the listener from `phpunit.xml.dist` and `composer.json`.

Extra points for removing the `lib` whilelist in `phpunit.xml.dist`. You already caught it in `composer.json`.

**Comment by kasperg** [comment]
This has been squashed, fixed and merged. Thanks @vakopian!

**Pull request closed without merging**